### PR TITLE
Update status of LIPs after Lisk v4 release

### PR DESCRIPTION
**Pull request by janhack**

### CODE REVIEW

File: proposals/lip-0003.md [hunk 1]
```diff
@@ -3,11 +3,12 @@ LIP: 0003
 Title: Uniform ordering of delegates list
 Author: Iker Alustiza <iker@lightcurve.io>
 Discussions-To: https://research.lisk.com/t/uniform-ordering-of-delegates-list/
-Status: Active
+Status: Replaced
 Type: Standards Track
 Created: 2018-08-17
 Updated: 2021-12-01
 Requires: 0022
+Superseded-By: 0057
```
Comment (ricott1) [inline proposals/lip-0003.md:11]: Should we be consistent and use either `Replaced` and `Replaced-by` or `Superseded` and `Superseded-by`?

Comment (ishantiw) [reply proposals/lip-0003.md:11]: I also agree with @ricott1

Comment (janhack) [reply proposals/lip-0003.md:11]: I follow LIP 0001 here and the header there https://github.com/LiskHQ/lips/blob/main/proposals/lip-0001.md#lip-header-preamble. I wouldn't change this now.

Comment (ricott1) [reply proposals/lip-0003.md:11]: sounds good, cbr

File: proposals/lip-0036.md [hunk 2]
```diff
@@ -2,11 +2,12 @@
 LIP: 0036
 Title: Introduce transaction properties moduleID and assetID
 Author: Shusetsu Toda <shusetsu@lightcurve.io>
-Status: Active
+Status: Replaced
 Type: Standards Track
 Created: 2020-09-16
 Updated: 2021-12-01
 Requires: 0028
+Superseded-By: 0069
```
Comment (ricott1) [inline proposals/lip-0036.md:10]: Maybe this should be `0068`

**Review by ricott1 — CHANGES_REQUESTED** [review]
One comment about superseding LIP

**Review by ishantiw — COMMENTED** [review]
Apart from @ricott1 comment, LGTM

File: proposals/lip-0036.md (hunk 2 above)
Comment (janhack) [reply proposals/lip-0036.md]: Changed as suggested.

**Review by ishantiw — APPROVED** [review]

**Review by ricott1 — APPROVED** [review]

**Pull request merged**